settings/checkstyle.xml

```diff
@@ -71,7 +71,7 @@
     <!-- Checks for Size Violations.                    -->
     <!-- See http://checkstyle.sf.net/config_sizes.html -->
     <module name="LineLength">
-      <property name="max" value="150" />
+      <property name="max" value="120" />
```

Comment:
It seems that editorconfig has `max_line_length` but IntelliJ does not support that. :(